hokusai/commands/namespace.py

```diff
@@ -25,7 +25,10 @@ def create_new_app_yaml(source_file, app_name):
       ('apiVersion', 'v1'),
       ('kind', 'Namespace'),
       ('metadata', {
-        'name': clean_string(app_name)
+        'name': clean_string(app_name),
+        'labels': {
+          'app-phase': 'review'
```

Comment:
potentially another use case for a `~/.hokusai.conf` config file. i would have a var there like `org=acme`, then this label can become `acme/app-phase: review` which should never conflict with any label k8s might automatically add to this resource in the future.